sdk/proto/phpfpm.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
+package f5.nginx.agent.sdk;
+
+import "gogo.proto";
+
+option go_package = "github.com/nginx/agent/sdk/v2/proto;proto";
+
+// PhpFpmDetails registers a phpfpm master process + child pools
+message PhpFpmDetails {
+  // PhpFpm process type (e.g. phpfpm)
+  string type = 1 [(gogoproto.jsontag) = "type"];
+  // System ID
+  string uuid = 2 [(gogoproto.jsontag) = "uuid"];
+  // PhpFpm ID
+  string local_id = 3 [(gogoproto.jsontag) = "local_id"];
+  // PhpFpm process name
+  string name = 4 [(gogoproto.jsontag) = "name"];
+  // PhpFpm process descpription
+  // Example: php-fpm: master process (/etc/php/7.4/fpm/php-fpm.conf)
+  string cmd = 5 [(gogoproto.jsontag) = "cmd"];
+  // Location of PhpFpm config path
+  // Example: /etc/php/7.4/fpm/php-fpm.conf
+  string conf_path = 6 [(gogoproto.jsontag) = "conf_path"];
+  // Location of PhpFpm binary path
+  string bin_path = 7 [(gogoproto.jsontag) = "bin_path"];
+  // PhpFpm version
+  string version = 8 [(gogoproto.jsontag) = "version"];
+  // PhpFpm version line
+  // Example: PHP 7.4.33 (fpm-fcgi) (built: Feb 14 2023 18:31:23)
+  string version_line = 9 [(gogoproto.jsontag) = "version_line"];
+  // PhpFpm process id
+  int32 pid = 10 [(gogoproto.jsontag) = "pid"];
+  // NGINX agent version
+  string agent = 11 [(gogoproto.jsontag) = "agent"];
+  // PhpFpm children process
+  repeated PhpFpmPool children = 12 [(gogoproto.jsontag) = "children"];
+  // PhpFpm master process total workers/children
+  int32 workers = 13 [(gogoproto.jsontag) = "workers"];
+  // PhpFpm process health
+  PhpFpmHealth health = 14 [(gogoproto.jsontag) = "health"];
```

Comment:
should the health be separated from the details?

Reply:
I thought of sending it out as part of php-information that will have meta details. Do you have any other thoughts on this?